### PR TITLE
ARUHA-77 Use real database EventType repository

### PR DESCRIPTION
**Pull request by tmuehl**
@v-stepanov @CyberDem0n @rcillo Please review

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/95" target="_blank">ReviewNinja

File: src/main/java/de/zalando/aruha/nakadi/config/WebConfig.java [hunk 1]
```diff
@@ -36,7 +38,7 @@ public TimeoutCallableProcessingInterceptor timeoutInterceptor() {
 
     @Bean
     public EventPublishingController eventPublishingController() {
```
Comment (v-stepanov) [inline src/main/java/de/zalando/aruha/nakadi/config/WebConfig.java]: I wonder why you decided to put `EventPublishingController` to `WebConfig`, not to `NakadiConfig`

File: src/main/java/de/zalando/aruha/nakadi/config/WebConfig.java [hunk 2]
```diff
@@ -35,11 +30,6 @@ public TimeoutCallableProcessingInterceptor timeoutInterceptor() {
     }
 
     @Bean
-    public EventPublishingController eventPublishingController() {
-        return new EventPublishingController(kafkaRepository, new InMemoryEventTypeRepository());
```
Comment (rcillo) [inline src/main/java/de/zalando/aruha/nakadi/config/WebConfig.java]: I completely missed this :sweat:

**Comment by rcillo** [comment]
:+1:

File: src/main/java/de/zalando/aruha/nakadi/config/WebConfig.java (hunk 2 above)
Comment (tmuehl) [inline src/main/java/de/zalando/aruha/nakadi/config/WebConfig.java]: No, no, you haven't missed anything. It was my technical dept. :)

**Pull request merged**